komga-webui/src/locales/en.json

```diff
@@ -230,37 +230,37 @@
     "book_number": "Book number: {name}",
     "book_series": "Series: {name}",
     "button_import": "Import",
-    "comicrack_preambule_html": "You can import existing ComicRack Reading Lists in <code>.cbl</code> format.<br/>Komga will try to match the provided series and book number with series and books in your libraries.",
+    "comicrack_preambule_html": "When import existing ComicRack Reading Lists in <code>.cbl</code> format<br/>Komga tries to match the provided series and book number with series and books in your libraries.",
     "field_files_label": "ComicRack Reading Lists (.cbl)",
     "import_read_lists": "Import Read Lists",
     "imported_as": "Imported as {name}",
-    "results_preambule": "Result of the import is shown below. You can also check the unmatched books for each provided file.",
-    "size_limit": "Size should be less than {size} MB",
```

Comment:
Unmarked/reflective error.

Reply:
what does that mean?

Reply:
You can't just cite "size" as a capacity that way.
-Here is a shirt.
answer
"The size is fitting". vs. "Size is fitting"